microsoft-edge/devtools-guide-chromium/storage/indexeddb.md

```diff
@@ -28,110 +28,133 @@ To view and change [IndexedDB](https://developer.mozilla.org/docs/Web/API/Indexe
+
+![Using a Snippet to interact with IndexedDB](./indexeddb-images/edit-from-snippet.png)
 
 
 <!-- ====================================================================== -->
 ## Delete IndexedDB data
 
-### Delete an IndexedDB key-value pair
+
+You can delete any of the following:
+* An IndexedDB key-value pair.
+* All key-value pairs in an object store.
+* An IndexedDB database.
+* All IndexedDB storage.
+
+These options are described below.
+
+
+#### Delete an IndexedDB key-value pair
 
 1. [View an IndexedDB object store](#view-indexeddb-data).
 
-1. Click the key-value pair that you want to delete.  DevTools highlights it to indicate that it is selected.
+1. Click the key-value pair that you want to delete.  DevTools highlights the key-value pair to indicate that it is selected:
 
-   ![Click a key-value pair in order to delete it](./indexeddb-images/storage-application-storage-indexeddb-notes_db-notes_os2.png)
+   ![A key-value pair item is selected in the object store view](./indexeddb-images/select-keyval.png)
 
-1. Press `Delete` or click **Delete Selected** (![Delete Selected](./indexeddb-images/delete-icon.png)).
+1. Press `Delete` on your keyboard or click the **Delete Selected** (![Delete Selected icon](./indexeddb-images/delete-icon.png)) button:
 
-   ![How the object store looks after the key-value pair has been deleted](./indexeddb-images/storage-application-storage-indexeddb-notes_db-notes_os-delete-selected.png)
+   ![The delete key value button in the toolbar of the object store view](./indexeddb-images/delete-keyval.png)
 
-### Delete all key-value pairs in an object store
+#### Delete all key-value pairs in an object store
 
 1. [View an IndexedDB object store](#view-indexeddb-data).
 
-   ![View an object store](./indexeddb-images/storage-application-storage-indexeddb-notes_db-notes_os-clear-object-store.png)
+1. Click the **Clear object store** (![Clear object store](./indexeddb-images/clear-icon.png)) button.
 
-1. Click **Clear object store** (![Clear object store](./indexeddb-images/clear-icon.png)).
+   ![The clear object store button in the toolbar of the object store view](./indexeddb-images/clear-object-store.png)
 
-### Delete an IndexedDB database
+#### Delete an IndexedDB database
 
 1. [View the IndexedDB database](#view-indexeddb-data) that you want to delete.
 
 1. Click **Delete database**.
 
-   ![The Delete database button](./indexeddb-images/storage-application-storage-indexeddb-notes_db-delete-database.png)
+   ![The Delete database button](./indexeddb-images/delete-database.png)
 
-### Delete all IndexedDB storage
+#### Delete all IndexedDB storage
 
-1. Open the **Clear storage** pane.
+1. In the sidebar of the **Application** tool, click **Storage**.
 
-1. Make sure that the **IndexedDB** checkbox is enabled.
+1. Scroll down to the **Storage** checkboxes and make sure that the **IndexedDB** checkbox is selected.
 
 1. Click **Clear site data**.
```

Comment:
```suggestion
1. Click the **Clear site data** button.
```
corp-level style guide says "don't state UI control type (unless doing so is helpful)", yet also states "don't assume users can view pngs".
so, plan: product-level style guide will say "always state the UI control type".